pkgs/tools/security/ghidra/build-extension.nix

```diff
@@ -26,9 +26,11 @@ let
       preBuild = ''
         # Set project name, otherwise defaults to directory name
         echo -e '\nrootProject.name = "${pname}"' >> settings.gradle
+        ${args.preBuild or ""}
       '';
 
-      gradleFlags = [ "-PGHIDRA_INSTALL_DIR=${ghidra}/lib/ghidra" ];
+      gradleBuildTask = args.gradleBuildTask or "buildExtension";
```

Comment:
```suggestion
```

Reply:
It is the default in the support file that the Ghidra docs specify as a required build file for all extension:
 https://github.com/NationalSecurityAgency/ghidra/blob/2517a3234822de74e05e642a55f79d8f3b4c5198/Ghidra/RuntimeScripts/Common/support/buildExtension.gradle#L18
 
 Unfortunately their internal extensions aren't bound by this convention

Reply:
I wish #315672 got merged before the refactor so we could get some more realistic users of the builder in nixpkgs.